sdk/storage/storage-queue/src/generated/src/storageClientContext.ts

```diff
@@ -20,14 +19,15 @@ export class StorageClientContext extends coreHttp.ServiceClient {
   /**
    * Initializes a new instance of the StorageClientContext class.
    * @param url The URL of the service account, queue or message that is the targe of the desired
-   * operation.
-   * @param [options] The parameter options
+   *            operation.
+   * @param options The parameter options
    */
-  constructor(url: string, options?: coreHttp.ServiceClientOptions) {
-    if (url == undefined) {
-      throw new Error("'url' cannot be null.");
+  constructor(url: string, options?: StorageClientOptionalParams) {
+    if (url === undefined) {
```

Comment:
`url == undefined` would handle both "null" and "undefined".

Is this a regression in the code-gen from autorest?

Reply:
Yes. This is a regression from code gen. Will remove it.